src/etc/inc/certs.inc

```diff
@@ -184,6 +185,11 @@ function ca_import(& $ca, $str, $key = "", $serial = "") {
 	/* Correct if child certificate was loaded first */
 	if (is_array($config['ca'])) {
 		foreach ($config['ca'] as & $oca) {
+			// check by serial number if CA already exists
+			$osn = cert_get_serial($oca['crt']);
+			if ($ca['refid'] <> $oca['refid'] && $serialNumber == $osn) {
```

Comment:
These comparisons each need their own set of parenthesis

Reply:
fixed